File: Basics/Nested-Loops/Lab/sum_of_two_numbers.py
interval_start = int(input())
interval_end = int(input())
magick_number = int(input())
combinations_counter = 0
is_combination_found = False

for x in range(interval_start, interval_end + 1):
    for y in range(interval_start, interval_end + 1):
        combination = x + y
        combinations_counter += 1

        if combination == magick_number:
            is_combination_found = True
            break

    if is_combination_found:
        break

if is_combination_found:
    print(f"Combination N:{combinations_counter} ({x} + {y} = {magick_number})")
else:
    print(f"{combinations_counter} combinations - neither equals {magick_number}")

File: JavaScript/Basics/0.6-Nested-Loops/Lab/sum-of-two-numbers.js
function sumOfTwoNumbers(input) {
    let start = Number(input.shift());
    let end = Number(input.shift());
    let m = Number(input.shift()); let c = 0;
    for (let x = start; x <= end; x++) {
        for (let y = start; y <= end; y++) {
            c++;
            if (x + y == m) {
                return `Combination N:${c} (${x} + ${y} = ${m})`;
            }
        }
    }
    return `${c} combinations - neither equals ${m}`;
}


console.log(sumOfTwoNumbers(["1",
"10",
"5"]));
console.log(sumOfTwoNumbers(["23",
"24",
"20"]));